eisenhower-matrix/src/main/webapp/script.js

```diff
@@ -158,29 +123,50 @@ async function createCal() {
     var SCOPES = "https://www.googleapis.com/auth/calendar";
     fetch("https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest") // Call the fetch function passing the url of the API as a parameter
     .then(function() {
+        //initializes api client
         gapi.load("client:auth2", initClient);
 
         function initClient() {
-            console.log("fetched google calendar api");
 
             gapi.client.init({
                 apiKey: API_KEY,
                 clientId: CLIENT_ID,
                 discoveryDocs: DISCOVERY_DOCS,
                 scope: SCOPES
             }).then(function () {
-                // Listen for sign-in state changes.
-                console.log(gapi.auth2.getAuthInstance().isSignedIn.get());
+                // Creates and adds new calendar that will contain scheduled tasks
                     var newCal = gapi.client.calendar.calendars.insert({
                         "summary": "Task Schedule",
                         "description": "Your tasks, scheduled by the Eisenhower Matrix."
                     }).then(function(response) {
+                        gapi.client.calendar.settings.get({setting:"timezone"}).then(
```

Comment:
We are using this snip too much, if you can check the branch I was using to create events, you will see that I started to create separate functions that way it's not necessary to repeat the code like.

Reply:
Which branch are you working on?

Reply:
get-events